### DISCUSSION TITLE
FVTT Mega 汉化包——建议贴

### DISCUSSION THREAD

**Discussion by hmqgg** [fvtt-cn/mega_chn#4] [综合讨论]
# Mega 汉化包
Mega 汉化包旨在翻译各种 FVTT 中文用户常用的 MOD、系统。如果有推荐加入 Mega 汉化包内的 MOD 或者系统，请在该贴中回复。

**Comment by hmqgg** [comment]
- [x] SmallTime
- [x] Active Auras
- [x] Polyglot
- [x] PDFoundry

**Reply by bnp800** [reply]
希望添加以下mod
- [x] Minimal UI
- [x] Monk's little details
- [x] Quick insert
- [ ] Combat numbers

**Reply by hmqgg** [reply]
目前主要加的是 PL 会接触到的，如果全部文本基本都是配置项这种 GM 接触就行的，目前可以先放置。

**Reply by hmqgg** [reply]
已加入 Quick Insert 和 Minimal UI

**Comment by xtlcme** [comment]
希望追加Monk全系列
- [ ] Monk's Active Tile Triggers
- [x] Monk's TokenBar
- [ ] Monk's Enhanced Journal
- [ ] Monk's Wall Enhancement
- [ ] Monk's Scene Navigation
- [ ] Monk's Hotbar Expansion
- [x] Monk's Little Details
- [x] Monk's Common Display

**Reply by hmqgg** [reply]
先安排了三个，另一部分的仅 GM 才会接触到的 MOD 还在考虑。

**Comment by TravelingK** [comment]
希望添加GURPS的系统
[GURPS 4th Edition Game Aid (Unofficial)
](https://foundryvtt.com/packages/gurps)

**Reply by TravelingK** [reply]
顺便[let's trade 5e ](https://github.com/KageJittai/lets-trade-5e)
这个mod只有几十个单词，非常简单

**Comment by bnp800** [comment]
希望加入 [swade tools](https://github.com/lipefl/swade-tools)

**Comment by hmqgg** [comment]
现阶段 [mega_chn](https://weblate.dickytwister.org/projects/mega_chn/) 还有大量未完工翻译。
要加新项目，应该会等到当前这些项目未翻译清零后再说。

**Reply by TravelingK** [reply]
现在就剩下几个游戏系统了吧，这几个规则我都不熟悉，帮不上忙。不如把[GUPRPS的系统](https://foundryvtt.com/packages/gurps)加进来，我来翻译

**Comment by bnp800** [comment]
希望加入ready to use cards https://gitlab.com/adrien.schiehle/ready-to-use-cards

极大地提高了卡牌系统的易用性

**Reply by hmqgg** [reply]
已添加。

**Comment by azurespear** [comment]
midi的专注状态在翻译后无法触发midi的自动专注管理

**Comment by baiyaojun** [comment]
希望添加mod【SWADE Immersive Macros (SWIM)】，以便提高FVTT-SWADE系统的游戏便利性

**Comment by lemonvama** [comment]
希望增加赛博朋克2020的系统汉化

**Reply by lemonvama** [reply]
如果可以的话，我可以把这个翻译了。毕竟是我要开的团。^^

**Comment by DrWuYe** [comment]
希望添加Cyberpunk RED，可以挂载到[Hosted Weblate](https://hosted.weblate.org/projects/?page=1&limit=100&sort_by=name)我来翻译，谢谢。

**Reply by hmqgg** [reply]
本来我翻译的部分在 Mega 里了，但是官方有 Crowdin，我把我翻译的部分放到官方的 Crowdin 了，你可以搜一下加入翻译。

**Comment by MeatMan258** [comment]
希望添加Shadowrun 6e，以及看到说Shadowrun 5e有加入，但实际使用没有生效，是我的设置问题吗？

**Reply by hmqgg** [reply]
因为 SR5e 虽然加入了，但是没有怎么翻译，所以基本无效。
可以加入 SR6e，但需要有人翻译才可以。

**Comment by jvgvghbn** [comment]
请问可以将『永い後日談のネクロニカ』(Nechronica) 添加到Hosted Weblate上吗？非常感谢！
